### PR TITLE
Updated SRM_creating_booking_silently with method to wait if booking …

### PR DESCRIPTION
**Pull request by JorgeFilipeDias**
Updated SRM_creating_booking_silently with method to wait if booking is created in the past

### CODE REVIEW

File: user-guide/Standard_Apps/SRM/Interacting_with_SRM_using_code/SRM_silent_actions/SRM_creating_booking_silently.md [hunk 1]
```diff
@@ -107,3 +107,40 @@ public class Script
 >
 > - In case a null service definition is provided, SRM will use a dynamically generated blank service definition.<!-- RN 30324 -->
 > - To add a resource without linking it to a node in the service definition, do not provide the *Id* attribute of the *Function* object.<!-- RN 30324 -->
+
+# Handling bookings with start date in the past
+
+When a booking is created with the method above and the start date is in the past is recomended to wait until the booking is fully started up before doing any other operation, otherwise some updates done by the start action may be overridden.
```
Comment (slc-peter-vanpoucke) [inline user-guide/Standard_Apps/SRM/Interacting_with_SRM_using_code/SRM_silent_actions/SRM_creating_booking_silently.md:113]: ... in the past it is recommended to ...

File: user-guide/Standard_Apps/SRM/Interacting_with_SRM_using_code/SRM_silent_actions/SRM_creating_booking_silently.md [hunk 2]
```diff
@@ -107,3 +107,40 @@ public class Script
...
+
+The following code can be used to wait:
```
Comment (slc-peter-vanpoucke) [inline user-guide/Standard_Apps/SRM/Interacting_with_SRM_using_code/SRM_silent_actions/SRM_creating_booking_silently.md:115]: S: ... to wait after *CreateNewBooking* successfully created the booking:

**Review by slc-peter-vanpoucke — CHANGES_REQUESTED** [review]

File: user-guide/Standard_Apps/SRM/Interacting_with_SRM_using_code/SRM_silent_actions/SRM_creating_booking_silently.md (hunk 1 above)
Comment (JorgeFilipeDias) [reply user-guide/Standard_Apps/SRM/Interacting_with_SRM_using_code/SRM_silent_actions/SRM_creating_booking_silently.md]: done

File: user-guide/Standard_Apps/SRM/Interacting_with_SRM_using_code/SRM_silent_actions/SRM_creating_booking_silently.md (hunk 2 above)
Comment (JorgeFilipeDias) [reply user-guide/Standard_Apps/SRM/Interacting_with_SRM_using_code/SRM_silent_actions/SRM_creating_booking_silently.md]: done

**Review by slc-peter-vanpoucke — APPROVED** [review]

**Comment by EdithLansens** [comment]
I've made some changes. Is this OK to merge, @JorgeFilipeDias?

**Comment by JorgeFilipeDias** [comment]
> I've made some changes. Is this OK to merge, @JorgeFilipeDias?

yes, that is ok

**Review by slc-peter-vanpoucke — APPROVED** [review]

**Review by EdithLansens — APPROVED** [review]

**Pull request merged**